src/encoded/static/components/browse/components/file-tables.js

```diff
@@ -412,6 +411,45 @@ export class RawFilesStackedTable extends React.PureComponent {
         this.state = {
             'mounted' : false
         };
+
+        this.memoized = {
+            groupedData: memoize(function(experimentSet){
+                const { experiments_in_set, replicate_exps } = experimentSet;
+                const experimentsGroupedByBiosample = expFxn.groupExperimentsByBiosampleRepNo(
+                    expFxn.combineWithReplicateNumbers(replicate_exps, experiments_in_set)
+                );
+                const allRawFiles = expFxn.allFilesFromExperimentSet(experimentSet, false);
+                return { experimentsGroupedByBiosample, allRawFiles };
+            })
+        };
+    }
+
+    analyticsImpression(){
+        const { experimentSet } = this.props;
+        const { allRawFiles, experimentsGroupedByBiosample } = this.memoized.groupedData(experimentSet);
+
+        const biosamples = [];
+        let exps = [];
+
+        experimentsGroupedByBiosample.forEach(function(expsWithBiosample){
+            const [ { biosample } ] = expsWithBiosample; // From first exp, grab out biosample (all exps should have same 1).
+            biosamples.push(biosample);
+            exps = exps.concat(expsWithBiosample);
+        });
+
+        setTimeout(function(){ // Wait for analytics initialization to complete.
+            analytics.impressionListOfItems(biosamples, null, "RawFilesStackedTable");
+            analytics.impressionListOfItems(exps, null, "RawFilesStackedTable");
+            analytics.impressionListOfItems(allRawFiles, null, "RawFilesStackedTable");
+            analytics.event("RawFilesStackedTable", "Mounted", { eventLabel: experimentSet.display_title });
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['experimentSet.display_title' is missing in props validation](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=4885443)